src/main/resources/liquibase/harness/generateChangelog/expectedChangeLog/cockroachdb/createFunction.sql

```diff
@@ -0,0 +1,4 @@
+INVALID TEST
+
+Functions are not supported by Cockroach DB
```

Comment:
cockroachdb v 22 supports user defined functions. Plz create separate ticket to unignore functions tests in all testcases

Reply:
As far as I can tell, cockroachdb supports functions only in v 22.2 and we are using 22.1 which still shows an error `You have attempted to use a feature that is not yet implemented.`